pages/common/immich-cli.md

```diff
@@ -0,0 +1,29 @@
+# immich-cli
+
+> Immich has a command line interface (CLI) that allows you to perform certain actions from the command line.
```

Comment:
```suggestion
```

IMO, it is redundant.

Reply:
Yes, it is. It's the "official" description.

Maybe instead of "... from the command line." we use "... from the terminal."